frontend/src/views/FieldAndSoil/FieldList/FieldList.tsx

```diff
@@ -18,9 +28,16 @@ const initialFieldFormData = {
   PreviousYearManureApplicationFrequency: '0',
   Comment: '',
   SoilTest: {},
-  Crops: [{}],
+  Crops: [],
 };
 
+const manureOptions = [
```

Comment:
These should be fetched from the backend, now that we're including tables. But, if you want to just get the styling in, I'll approve and you can make a followup PR.